web-frame-error/index.html

```diff
@@ -0,0 +1,112 @@
+<!--
+  @license
+  Copyright (c) 2018-2020 THREEANGLE SOFTWARE SOLUTIONS SRL
+  Available under MIT license webFrame/LICENSE
+-->
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta content="width=device-width, initial-scale=1.0" name="viewport">
+  <meta content="ie=edge" http-equiv="X-UA-Compatible">
+  <title> Runtime Error!</title>
+  <base href="/">
+  <style>
+    body {
+      display: grid;
+      font-family: "-apple-system", "BlinkMacSystemFont", "Segoe UI", "Roboto", "Oxygen-Sans", "Ubuntu", "Cantarell", "Helvetica Neue", system-ui, sans-serif;
+      margin: 1em;
+      min-height: 75vh;
+      place-content: center stretch;
+    }
+
+    h1 {
+      border-bottom: 1px double currentColor;
+    }
+
+    pre {
+      overflow: auto;
+      white-space: pre-wrap;
+    }
+  </style>
+</head>
+<body>
+  <div class="error-page">
+    <h1>Runtime Error!</h1>
+    <p>Please contact your administrator for more information.</p>
+    <p><a href="/">Go to homepage</a></p>
+    <details>
+      <summary>View details</summary>
+      <div id="error-details"></div>
+    </details>
+  </div>
+  <script>
+    // @ts-check
+    const LAST_RUNTIME_ERROR_KEY = 'lastRuntimeError';
+
+    /**
+     * @return { Error | undefined }
+     */
+    function getErrorInfo() {
+      /** @type { Partial<Error> | unknown } */
+      let errorDetails;
+      try {
+        const errorInfoString = /** @type { string } */
+          window.sessionStorage.getItem(LAST_RUNTIME_ERROR_KEY);
+        errorDetails = JSON.parse(errorInfoString);
+      } catch (err) {
+        // Nothing to do.
+      }
+
+      if (errorDetails && errorDetails.hasOwnProperty('name')) {
+        /** @type { Partial<Error> } */
+        const errorInfo = {
+          name: 'Unknown Error',
+          message: undefined,
+          stack: undefined,
+        };
+
+        if (errorDetails.hasOwnProperty('name')) {
+          errorInfo.name = `Name: ${errorDetails.name}`;
+        }
+        if (errorDetails.hasOwnProperty('message')) {
+          errorInfo.message = `Message: ${errorDetails.message}`;
+        }
+        if (errorDetails.hasOwnProperty('stack')) {
+          errorInfo.stack = errorDetails.stack;
+        }
+        return errorInfo;
+      }
+
+      return undefined;
+    }
+
+    /** @type { {errorInfo: Partial<Error>} } */
+    const ERROR_PAGE_DATA = {};
+    ERROR_PAGE_DATA.errorInfo = getErrorInfo();
+
+    const detailsContainerEl = /** @type { HTMLDivElement } */
+      document.getElementById('error-details');
+
+    if (ERROR_PAGE_DATA.errorInfo) {
+      const errorNameEl = /** @type { HTMLHeadingElement } */
+        document.createElement('h2');
```

Comment:
add proper identation for all

Reply:
Fixed. See https://github.com/3angleTech/webFrame/pull/140#discussion_r439670762